snowpack/src/types/snowpack.ts

```diff
@@ -11,15 +11,20 @@ export type DeepPartial<T> = {
 
 export type EnvVarReplacements = Record<string, string | number | true>;
 
-export type SnowpackBuiltFile = {code: string | Buffer; map?: string};
+export type FileContents = string | Buffer
+
+export type SnowpackBuiltFile = {
+  contents: FileContents;
```

Comment:
I would keep `code` but add a jsdoc above it with a `@deprecated` tag

```suggestion
  /**
   *  @deprecated `.code` is deprecated, use `.contents` instead.
   */
  code: FileContents
  contents: FileContents;
```

Reply:
I think @FredKSchott normally likes to drop it from the typings but since this is exposed and could potentially be used, I could see it being considered "breaking". I normally add the `@deprecated` tag. (I actually had it and deleted it)

Reply:
It will help people to migrate to the new code, I think VS Code highlights code that is deprecated in types now

Reply:
+1 in favor of @stramel’s approach: just removing `code` entirely internally (but still keeping it as an external API for plugins to prevent breaking changes)